packages/financial-templates-lib/src/logger/PagerDutyTransportV2.ts

```diff
@@ -0,0 +1,49 @@
+// This transport enables winston logging to send messages to pager duty v2 api.
+import Transport from "winston-transport";
+import { event } from "@pagerduty/pdjs";
+
+type TransportOptions = ConstructorParameters<typeof Transport>[0];
+export type Severity = "critical" | "error" | "warning" | "info";
+
+interface Config {
+  integrationKey: string;
+  customServices?: { [key: string]: string };
+}
+
+export class PagerDutyTransportV2 extends Transport {
+  private readonly integrationKey: string;
+  private readonly customServices: { [key: string]: string };
+  constructor(winstonOpts: TransportOptions, { integrationKey, customServices = {} }: Config) {
+    super(winstonOpts);
+    this.integrationKey = integrationKey;
+    this.customServices = customServices;
```

Comment:
Is this unused? Does this mean this transport will not allow us to route to multiple services in the same bot?

Reply:
i dont really understand how this is used, but in the pd2 api, there is no field for `serviceId`.  In v1 we do: `const serviceId = this.customServices[info.notificationPath] ?? this.defaultServiceId;`.   In the v2 api theres maybe something similar with `payload.source=The unique location of the affected system, preferably a hostname or FQDN.` but i set this as the bot identifier.  

Take a look at the payload shape for v2 api as its quite different from v1: https://developer.pagerduty.com/docs/ZG9jOjExMDI5NTgx-send-an-alert-event